### PR TITLE
Add data source for WAF Packages

### PR DESCRIPTION
**Pull request by XaF**

### CODE REVIEW

File: cloudflare/data_source_waf_packages.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+func dataSourceCloudflareWAFPackages() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceCloudflareWAFPackagesRead,
+
+		Schema: map[string]*schema.Schema{
+			"zone_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
```
Comment (patryk) [inline cloudflare/data_source_waf_packages.go]: This field is required, yet not documented.

Comment (XaF) [reply cloudflare/data_source_waf_packages.go]: Fixed!

**Comment by patryk** [comment]
Thanks!

**Comment by XaF** [comment]
Thank you! :)

**Pull request merged**